uniflow/op/model/model_server.py

```diff
@@ -621,44 +613,38 @@ def __init__(
             import boto3
 
             super().__init__(prompt_template, model_config)
-            self._model_config = BedrockModelConfig(**self._model_config)
 
             # If user specifies profile in model config, use that profile
             if "aws_profile" in model_config:
-                aws_profile = self._model_config.aws_profile
-                session = boto3.Session(profile_name=aws_profile)
+                aws_profile = model_config["aws_profile"]
+                self.session = boto3.Session(profile_name=aws_profile)
             # Otherwise if the user specifies credentials directly in the model config, use those credentials
-            elif (
-                self._model_config.aws_access_key_id
-                and self._model_config.aws_secret_access_key
+            elif model_config.get("aws_access_key_id") and model_config.get(
+                "aws_secret_access_key"
             ):
-                session = boto3.Session(
-                    aws_access_key_id=self._model_config.aws_access_key_id,
-                    aws_secret_access_key=self._model_config.aws_secret_access_key,
-                    aws_session_token=self._model_config.aws_session_token,
+                self.session = boto3.Session(
+                    aws_access_key_id=model_config["aws_access_key_id"],
+                    aws_secret_access_key=model_config["aws_secret_access_key"],
+                    aws_session_token=model_config.get("aws_session_token"),
```

Comment:
nit: can we use .get to access the dict for consistency to improve readability.

Reply:
Sure! Will do